src/ServerService.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/**
+ * @package Wsdl2PhpGenerator
+ */
+namespace Wsdl2PhpGenerator;
+
+use Wsdl2PhpGenerator\PhpSource\PhpClass;
+use Wsdl2PhpGenerator\PhpSource\PhpDocComment;
+use Wsdl2PhpGenerator\PhpSource\PhpDocElementFactory;
+use Wsdl2PhpGenerator\PhpSource\PhpFunction;
+use Wsdl2PhpGenerator\PhpSource\PhpVariable;
+
+/**
+ * Service represents the service in the wsdl, from the server side
+ *
+ * @package Wsdl2PhpGenerator
+ * @author Thierry Blind <thierryblind@msn.com>
+ * @license http://www.opensource.org/licenses/mit-license.php MIT License
```

Comment:
Please remove this. I consider `wsdl2phpgenerator` a community project and I see no need to put names on individual classes.